end_app/weather_app/entities.py

```diff
@@ -103,25 +103,143 @@ def from_raw_forecast_data(cls, forecast_data):
 @dataclass
 class AirPollutionData:
     aqi: int
-    co: int
-    no: int
-    no2: int
-    o3: int
-    so2: int
-    pm2_5: int
-    pm10: int
-    nh3: int
+    co: float
+    no: float
+    no2: float
+    o3: float
+    so2: float
+    pm2_5: float
+    pm10: float
+    nh3: float
 
     @classmethod
     def from_raw_air_pollution_data(cls, air_pollution_data):
         return cls(
             aqi=air_pollution_data["list"][0]["main"]["aqi"],
-            co=air_pollution_data['list'][0]['components']["co"],
-            no=air_pollution_data['list'][0]['components']["no"],
-            no2=air_pollution_data['list'][0]['components']["no2"],
-            o3=air_pollution_data['list'][0]['components']["o3"],
-            so2=air_pollution_data['list'][0]['components']["so2"],
-            pm2_5=air_pollution_data['list'][0]['components']["pm2_5"],
-            pm10=air_pollution_data['list'][0]['components']["pm10"],
-            nh3=air_pollution_data['list'][0]['components']["nh3"],
+            co=round(air_pollution_data['list'][0]['components']["co"], 1),
+            no=round(air_pollution_data['list'][0]['components']["no"], 1),
+            no2=round(air_pollution_data['list'][0]['components']["no2"], 1),
+            o3=round(air_pollution_data['list'][0]['components']["o3"], 1),
+            so2=round(air_pollution_data['list'][0]['components']["so2"], 1),
+            pm2_5=round(air_pollution_data['list'][0]['components']["pm2_5"], 1),
+            pm10=round(air_pollution_data['list'][0]['components']["pm10"], 1),
+            nh3=round(air_pollution_data['list'][0]['components']["nh3"], 1),
         )
+
+    def get_scale_and_qualitative(self, value):
+        if value == 'aqi':
+            return '0-500', None
```

Comment:
tu potrzeba wytłumaczyć o co tu chodzi to takie magic stringi i magic numbery co nie bardzo coś mówią